pkg/admission/pluginconfig_webhook.go

```diff
@@ -21,6 +21,12 @@ import (
 	"github.com/cloudoperators/greenhouse/pkg/helm"
 )
 
+// pluginsAllowedInCentralCluster is a list of plugins that are allowed to be installed in the central cluster.
+// TODO: Make this configurable on plugin level (AdminPlugin discussion) instead of maintaining a list here.
+var pluginsAllowedInCentralCluster = []string{
+	"alerts", "doop", "service-proxy", "teams2slack",
```

Comment:
Should we add `service-proxy` and `kubeconfig-generator` to the list?

Reply:
done